core/src/main/java/software/amazon/awssdk/util/DateUtils.java

```diff
@@ -181,32 +126,53 @@ public static String formatIso8601Date(DateTime date) {
      *            The date string to parse.
      *
      * @return The parsed Date object.
+     * @deprecated use {@link #parseRfc1123Date(String)}. RFC-1123 updates RFC-822 changing the year from two digits to four.
```

Comment:
Gotcha; since we switched over to the `parseRfc1123Date` let's just remove this one